core/tenant/secret.go

```diff
@@ -8,10 +8,6 @@ const (
 	SecretStorageKey    = "STORAGE"
 	SecretSchedulerAuth = "SCHEDULER_AUTH"
```

Comment:
Can you add the alert secret to be a predefined secret as well, doesn't have much significance but a place where we need what all secrets are part of infra setup.